content/3_Silver/Greedy.mdx

```diff
@@ -102,29 +102,45 @@ export const problems = {
 			'571',
 			'Easy',
 			false,
-			[],
+			['Greedy', 'Sorting', 'Ordered Set'],
 			'usaco-571'
 		),
-		new Problem('Gold', 'High Card Low Card', '573', 'Normal', true, [], ''),
 		new Problem(
-			'Silver',
-			'Berry Picking',
-			'990',
+			'Gold',
+			'High Card Low Card',
+			'573',
+			'Easy',
+			true,
+			['Greedy', 'Sorting', 'Ordered Set'],
+			''
+		),
+		new Problem(
+			'CSA',
+			'Sure Bet',
+			'sure-bet',
 			'Normal',
 			false,
-			[],
-			'usaco-990'
+			['Greedy', 'Sorting', 'Two Pointers'],
```

Comment:
Should we stick to using "Two Pointers" or "2P" as the tag name?

Reply:
let's go w/ 2P